### ISSUE TITLE
Request to add OCI helm registry support

### ISSUE THREAD

**Issue by LinuxSuRen** [supabase-community/supabase-kubernetes#56]
# Feature request

I didn't find the helm chart registry from the document. I'd like to help to push the helm chart into an OCI registry. Please let me know if the maintainer willing to have it.

## Is your feature request related to a problem? Please describe.

None.

## Describe the solution you'd like

Put the helm chart into ghcr.io and docker.io

## Describe alternatives you've considered

None.

## Additional context

See also https://helm.sh/docs/topics/registries/

**Comment by drpsyko101** [comment]
Related to #55, I'm not sure whether is it a good idea to push to OCI now or if should we develop the chart until it becomes production-ready.

**Comment by LinuxSuRen** [comment]
Thanks for your response. IMO, it is also reasonable to have an easier version to let users to have a try.

**Comment by drpsyko101** [comment]
I agree with that. Feel free to voice your opinion in the PR and #53 on how should we move forward with the chart deployment.

**Comment by luizfelmach** [comment]
Hey,

Currently, Helm already supports the repository published via gh-pages at https://supabase-community.github.io/supabase-kubernetes, which allows it to be used normally with: `helm repo add supabase <repo>`

Given this, I’d like to understand whether it still makes sense to maintain an OCI Helm registry, or if—at least for now—the gh-pages repository alone would be sufficient.

**Comment by LinuxSuRen** [comment]
It's enough for me. So, I think this issue can be closed.
